releasenotes/notes/conservative-commutation-checking-b728e7b6e1645615.yaml

```
@@ -1,18 +1,19 @@
 ---
 fixes:
   - |
-    Commutation relations of :class:`~.circuit.Instruction`\ s with float-only ``params``
-    were eagerly cached by the :class:`.CommutationChecker`, using the ``params`` as key to
-    query the relation. This could lead to faulty results, if the instruction's definition
-    depended on additional information that just the :attr:`~.circuit.Instruction.params`
-    attribute, such as e.g. the case for :class:`.PauliEvolutionGate`.
-    This behavior is now fixed, and the commutation checker only conservatively caches
-    commutations for Qiskit-native standard gates. This can incur a performance cost if you were
-    relying on your custom gates being cached, however, we cannot guarantee safe caching for
-    custom gates, as they might rely on information beyond :attr:`~.circuit.Instruction.params`.
+    Previously, the :class:`.CommutationChecker` eagerly cached the commutation relations of
+    :class:`~.circuit.Instruction`\ s with  float-only ``params`` as key to query the relation.
+    This could lead to incorrect results if the instruction's definition
+    depended on additional information beyond just the :attr:`~.circuit.Instruction.params`
+    attribute, as in the case of :class:`.PauliEvolutionGate`.
```

Just a suggestion to point out it could be any other gate too 
```suggestion
    attribute, as, for example, in the case of :class:`.PauliEvolutionGate`.
```